debian/marquee/nginx/sites/html.spec.whatwg.org.conf

```diff
@@ -86,131 +86,833 @@ server {
     }
-    location = /the-th-element.html {
+    location = /multipage/aside.html {
+        return 301 /multipage/sections.html;
+    }
+    location = /multipage/association-of-controls-and-forms.html {
+        return 301 /multipage/form-control-infrastructure.html;
+    }
+    location = /multipage/attributes.html {
+        return 301 /multipage/dom.html;
+    }
+    location = /multipage/attributes-common-to-form-controls.html {
+        return 301 /multipage/form-control-infrastructure.html;
+    }
+    location = /multipage/attributes-common-to-td-and-th-elements.html {
         return 301 /multipage/tables.html;
     }
-    location = /the-track-element.html {
+    location = /multipage/audio.html {
         return 301 /multipage/media.html;
     }
-    location = /tokenization.html {
-        return 301 /multipage/parsing.html;
+    location = /multipage/b.html {
+        return 301 /multipage/text-level-semantics.html;
     }
-    location = /webappapis.html {
-        return 301 /multipage/webappapis.html;
+    location = /multipage/base.html {
+        return 301 /multipage/semantics.html;
     }
-
-    # Multipage; mostly for redirects from W3C TR/html and other older forks
-    location = /multipage/association-of-controls-and-forms.html {
-        return 301 /multipage/form-control-infrastructure.html;
+    location = /multipage/bdi.html {
+        return 301 /multipage/text-level-semantics.html;
+    }
+    location = /multipage/bdo.html {
+        return 301 /multipage/text-level-semantics.html;
+    }
+    location = /multipage/blockquote.html {
+        return 301 /multipage/grouping-content.html;
+    }
+    location = /multipage/body.html {
+        return 301 /multipage/sections.html;
+    }
+    location = /multipage/br.html {
+        return 301 /multipage/text-level-semantics.html;
+    }
+    location = /multipage/button.html {
+        return 301 /multipage/form-elements.html;
     }
     location = /multipage/button.button.html {
         return 301 /multipage/form-elements.html;
     }
+    location = /multipage/button.reset.html {
+        return 301 /multipage/form-elements.html;
+    }
     location = /multipage/button.submit.html {
         return 301 /multipage/form-elements.html;
     }
+    location = /multipage/caption.html {
+        return 301 /multipage/tables.html;
+    }
     location = /multipage/changes.html {
         return 301 /multipage/introduction.html;
     }
+    location = /multipage/cite.html {
+        return 301 /multipage/text-level-semantics.html;
+    }
+    location = /multipage/code.html {
+        return 301 /multipage/text-level-semantics.html;
+    }
+    location = /multipage/col.html {
+        return 301 /multipage/tables.html;
+    }
+    location = /multipage/colgroup.html {
+        return 301 /multipage/tables.html;
+    }
+    location = /multipage/command.html {
+        return 301 /multipage/obsolete.html;
+    }
+    location = /multipage/command.checkbox.html {
```

Comment:
Do you know what was at these command URLs? I'm unsure whether they should go to obsolete or to https://html.spec.whatwg.org/#commands

Reply:
Are those possibly related to this section? https://web.archive.org/web/20150317081302/https://html.spec.whatwg.org/multipage/forms.html#commands